### ISSUE TITLE
Move Snyk Scripts from k/test-infra to k/sig-security/sig-security-tooling

### ISSUE THREAD

**Issue by PushkarJ** [kubernetes/sig-security#60]
As we learnt from https://github.com/kubernetes/test-infra/pull/26896#discussion_r932628360 it is possible for prow to pick up shell scripts outside of `k/test-infra`.

Moving the script that is present here: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml#L29-L87 into a dedicated shell script with name `build-deps-and-release-images.sh` under `k/sig-security/sig-security-tooling/scanning` would be great for faster iteration and approvals and reviews can be managed within SIG Security members. 

/sig security
/good-first-issue
/help

**Comment by lakshya8066** [comment]
Hi @PushkarJ is this issue open for contribution?

**Comment by PushkarJ** [comment]
Yes the PR blocking it is merged. So you can work on this if you'd like

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/test-infra/pull/34431

**Comment by mtardy** [comment]
> [kubernetes/test-infra#34431](https://github.com/kubernetes/test-infra/pull/34431)

Indeed, thanks Ben for the note, it's be done by above issue.

/close
